lib/modules/begroot-widgets/index.js

```diff
@@ -183,6 +183,16 @@ module.exports = {
      const superLoad = self.load;
 
      self.load = function (req, widgets, callback) {
+       req.data.ideas = req.data.ideas.map((idea) => {
+         idea.showOriginalIdeaLink = true;
```

Comment:
You can also use a short expression: `idea.showOriginalIdeaLink = !!idea.extraData.hideOriginalIdeaLink;`

Reply:
You're right, that's a lot nicer. Thanks!